BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/security/service/SecurityService.java

```diff
@@ -77,12 +77,8 @@ public User getUser(long userId) {
     public UserData signIn(String login, String password) {
         Optional<User> user = userRepository.findByLoginOrEmail(login, login);
 
-        if (!user.isPresent()) {
-            throw new SecurityException(format("Could find an user register with email or login '%s'", login));
-        }
-
-        if (!securityUtil.checkPassword(user.get().getPasswordDigest(), password)) {
-            throw new SecurityException(format("Given password do not match for user '%s'", login));
+        if (!user.isPresent() || !securityUtil.checkPassword(user.get().getPasswordDigest(), password)) {
```

Comment:
I don't think we should merge these two. I think the intention is to have separated error messages.

Reply:
It's good practice not to share any information in error messages which can lead a hacker knowing if an email is correct or not.

Reply:
👍